python/cudf/cudf/tests/test_orc.py

```diff
@@ -634,6 +634,10 @@ def test_orc_write_statistics(tmpdir, datadir, nrows):
             stats_max = file_stats[0][col]["maximum"]
             actual_max = gdf[col].max()
             assert normalized_equals(actual_max, stats_max)
+        if "number_of_values" in file_stats[0][col]:
```

Comment:
There's a C++ ORC test that checks `number_of_values`.
I think it does not catch the issue because there's not enough rows to require block reduction, is that correct?

Reply:
Yeah that one is only 9 rows. This bug would appear when total number of rows > num rows in one rowgroup